### DISCUSSION TITLE
support poster output for accademic poster presentation

### DISCUSSION THREAD

**Discussion by mrymam** [marp-team/marp#419] [Ideas] [answered]
Marp is a powerful tool for presentation slide. I use marp for accademic presentation now! Thank you!

Since poster is style of 2 column and 1 page,  marp cannot make poster material.
But both of poster and slide have same structure that consist of some sections. 

Do you have plan to support poster style output ? 
Or should I use custom css template instead?

**Comment by yhatt** [comment, marked as answer]
Marp is basically designed for presentation slides, so Marp team would not take some action to support other purpose than Markdown presentation.

However, we do never limit user's creativity too. Actually, I know our community has made a custom theme CSS for the poster :)

> I tried making an academic poster in Marp [https://marp.app](https://t.co/wqUy2pLfzy) - it was super convenient to use Markdown, figures update automatically when changed, can write latex inline, etc. Took some CSS wrangling but even as someone w/ almost no web dev experience it wasn't too bad.
>
> _&mdash; https://twitter.com/bpedigod/status/1514337012874194955_

:arrow_right: https://github.com/neurodata/marp-poster-template

<img src="https://raw.githubusercontent.com/neurodata/marp-poster-template/main/examples/pedigo-naisys-2022/pedigo-naisys-2022.png" width="300" /> <img src="https://raw.githubusercontent.com/neurodata/marp-poster-template/main/examples/athey-nebec-2022/nebec.png" width="300" />

**Reply by mrymam** [reply]
Thank you for answer!
I will create own custome theme CSS with reference to suggested ones.
